chunky3d/chunky.py

```diff
@@ -567,6 +571,26 @@ def _update_memory_block(self, idx, d):
         else:
             self._grid[idx][:] = d
 
+    def _update_grid(self):
+        """In case the underlying `_memory_blocks` and `_grid_mask` change, this method updates the `_grid`."""
+        chunks = self.chunks
+        grid = {}
+        for i, j, k in np.argwhere(self.grid_mask != Sparse.EMPTY_GRID_VALUE):
+            idx = self._grid_mask[i, j, k]
+            envelope = 0
```

Comment:
Is this intentional? The access to `dense_data` below would be simplified if this variable disappeared.
Or was is supposed to be != 0 in special cases?

Reply:
Added the missing parameter.

Reply:
OK, but now this assignment needs to be removed.